modular_zubbers/code/modules/scents/rot.dm

```diff
@@ -12,13 +12,14 @@
 	if(!istype(T) || T.planetary_atmos || T.return_air().return_pressure() > (WARNING_HIGH_PRESSURE - 10))
 		return
 	var/datum/gas_mixture/stank = new
-	ADD_GAS(/datum/gas/miasma, stank.gases)
-	stank.gases[/datum/gas/miasma][MOLES] = ROT_MOLES
+	if(owner)
+		ADD_GAS(/datum/gas/scent/cadaverine, stank.gases)
+		stank.gases[/datum/gas/scent/cadaverine][MOLES] = ROT_MOLES
+	ADD_GAS(/datum/gas/scent/putrescine, stank.gases)
```

Comment:
You sure we should be stanking both cadaverine and putrescine here?

Reply:
it's intentional to follow up with making this a unique thing for the mortician later.